api/tests/unit/environments/dynamodb/test_unit_dynamodb_environment_v2_wrapper.py

```diff
@@ -0,0 +1,85 @@
+import os
+import uuid
+
+import boto3
+import pytest
+from moto import mock_dynamodb
+from mypy_boto3_dynamodb.service_resource import Table
+from pytest_django.fixtures import SettingsWrapper
+
+from environments.dynamodb.dynamodb_wrapper import DynamoEnvironmentV2Wrapper
+from environments.models import Environment
+from features.models import Feature
+from util.mappers.dynamodb import map_environment_to_environment_v2_document
+
+
+@pytest.fixture()
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+    os.environ["AWS_DEFAULT_REGION"] = "eu-west-2"
+
+
+@pytest.fixture()
+def dynamodb(aws_credentials):
+    # TODO: move all wrapper tests to using moto
```

Comment:
I've left this as a todo here since I don't want to bloat this PR with extra work, moving other tests and fixtures around.